cg/constants.py

```diff
@@ -24,6 +24,35 @@
     "LymphoMATIC",
     "other (specify in comment field)",
 )
+DEFAULT_CAPTURE_KIT = "agilent_sureselect_cre.v1"
+WGS_CAPTURE_KIT = "twistexomerefseq_9.1_hg19_design.bed"
```

Comment:
I think this could be replace this with default capture above. That is use DEFAULT_CAPTURE_KIT throughout the code base instead of WGS_CAPTURE_KIT